proto/backend.proto

```diff
@@ -53,32 +79,26 @@ message DataQuery {
   string refId = 1;
   int64 maxDataPoints = 2;
   int64 intervalMS = 3;
-  TimeRange timeRange = 4; // NOTE: this moved from QueryRequest in v1
-  bytes json = 5; // Arbitrary structure (model?  modelJson in v1)
+  TimeRange timeRange = 4;
+  bytes json = 5;
```

Comment:
Would suggest renaming this to `modelJSON` or `jsonModel`.

Reply:
linting tools hate lowercase `json`

Reply:
It's in generated code. But what do you suggest?

Reply:
`modelJSON` or `queryModelJSON`

Reply:
And what about health check which currently have `jsonDetails`?

Reply:
detailJSON?